client/src/Views/Home/Prospects/Prospects.js

```diff
@@ -1,16 +1,176 @@
-import React from "react";
-import { Grid, Typography, withStyles } from "@material-ui/core";
+import React, { Component } from "react";
+import { Grid, withStyles, Typography } from "@material-ui/core";
+import { Route } from "react-router-dom";
+import ProspectSidebar from "./ProspectSidebar";
+import ProspectDashboardHeader from "./ProspectDashboardHeader";
+import Box from '@material-ui/core/Box';
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableHead from '@material-ui/core/TableHead';
+import TableRow from '@material-ui/core/TableRow';
```

Comment:
I found you can keep all these on one line like:

`import { Box, Table, TableBody ... } from "@material-ui/core"`

Plus we're supposed to be using double-quotes by default in Prettier :)